sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/sources/ConsoleWriter.scala

```diff
@@ -56,7 +58,7 @@ trait ConsoleWriter extends Logging {
     println("-------------------------------------------")
     // scalastyle:off println
     spark
-      .createDataFrame(spark.sparkContext.parallelize(rows), schema)
+      .createDataFrame(rows.toList.asJava, schema)
```

Comment:
Change here to avoid triggering new distributed job.

Reply:
this fix should go into 2.3 branch. thanks for catching this.

Reply:
OK, I will create a separate PR for this small fix.